apps/web-giddh/src/app/home/components/cr-dr-list/cr-dr-list.component.ts

```diff
@@ -179,6 +179,7 @@ export class CrDrComponent implements OnInit, OnDestroy {
     public showGiddhDatepicker(element: any): void {
         if (element) {
             this.dateFieldPosition = this.generalService.getPosition(element.target);
+            this.dateFieldPosition.y = element.clientY + 20; // 20 px for inputbox height
```

Comment:
If you think this will solve issue on all pages, put this in function so that we don't have to do this in every component.